ge0/ge0_tests/parser_tests.cpp

```diff
@@ -312,4 +312,44 @@ UNIT_TEST(OtherPrefixes)
   TestSuccess("https://omaps.app/Byqqqqqqqq", 45, 0, 4.25, "");
   TestFailure("https://omaps.app/Byqqqqqqq");
 }
+
+UNIT_TEST(PlainCoordinateUrl_Valid)
```

Comment:
The current ge0 format stores and passes zoom level, so opened links can show roughly the same picture on the screen as it was when the link was created. This is very handy because the best zoom is already chosen at the moment of sharing the link.

There are different ways how to support zoom level in links, which one would be the best (compact, readable)? Maybe `/lat,lon,zoom/name` ? It likely also should be optional, and support floating-point format for better precision, right?